### ISSUE TITLE
présentation de airflow & comparaison avec luigi

### ISSUE THREAD

**Issue by garaud** [pybdx/meetup-hub#13]
# Présentation de airflow et comparaison avec lugi

* Damien
* **à construire**
* Je suis intéressé par le sujet mais aucune expérience en airflow
* Le type de format (court, long, *live coding*, atelier, autre, etc.)
* Je veux bien aider à mieux préciser le sujet & relire

En option, vous pouvez ajouter :

* Le type d'audience : débutant·e, intermédiaire, expert·e **todo**
* Disponibilité(s) **todo**

**Comment by josselinauguste** [comment]
Il y a un petit nouveau aussi : https://www.prefect.io/

**Comment by garaud** [comment]
Exact, je l'ai vu passer il y a quelques semaines en me disant "hey, faudrait que j'essaie"
